book/src/ledger-replication.md

```diff
@@ -66,75 +65,85 @@ mining proof claim has to contain to be valid for a reward.
 proof.
 6. NUM\_CHACHA\_ROUNDS: Number of encryption rounds performed to generate
 encrypted state.
+7. NUM\_SLOTS\_PER\_TURN: Number of slots that define a single storage epoch or
+a "turn" of the PoRep game.
 
 ### Validator behavior
 
-1. Validator joins the network and submits a storage validation capacity
-transaction which tells the network how many proofs it can process in a given
-period defined by NUM\_KEY\_ROTATION\_TICKS.
-2. Every NUM\_KEY\_ROTATION\_TICKS the validator stores the PoH value at that
-height.
-3. Validator generates a storage proof confirmation transaction.
-4. The storage proof confirmation transaction is integrated into the ledger.
-6. Validator responds to RPC interfaces for what the last storage epoch PoH
-value is and its slot.
+1. Validators join the network and begin looking for replicator accounts at each
+storage epoch/turn boundary.
+2. Every turn, Validators sign the PoH value at the boundary and use that signature
+to randomly pick proofs to verify from each storage account found in the turn boundary.
+This signed value is also submitted to the validator's storage account and will be used by
+replicators at a later stage to cross-verify.
+3. Every `NUM_SLOTS_PER_TURN` slots the validator advertises the PoH value. This is value
+is also served to Replicators via RPC interfaces.
+4. For a given turn N, all validations get locked out until turn N+2 (a gap of 1 turn/epoch).
```

Comment:
What do you mean exactly by locked out until N+2? They cannot claim a reward until then, right?

Reply:
It's more like N+3 right:

| storage proofs (N) | proof validations (N+1) | replicator challenge (N+2) | reward claiming (N+3) |

Reply:
Yeah I need to see how to clarify this. I'll try here and if its good enough I'll promote it to the book.

The validators are watching banks at turn boundaries and advertising the latest turn boundary slot. 

At every turn boundary, the PoRep game can be in 4 different stages. 
- Proof submission stage
- Proof verification stage
- Proof challenge stage
- Reward collection stage

Just saw other comment. Yes it's more like N+3 
```
| storage proofs (N) | proof validations (N+1) | replicator challenge (N+2) | reward claiming (N+3) |
```

For every reported turn boundary on a replicator, I'm going to make the replicator ask for its account and run the Replicator Challenge and Reward Claiming stage.
For turn boundaries on valiators I'm going to make them run the Proof Validation Stage and the Reward Stage on that boundary. 

Hopefully this way the PoRep game keeps ticking along. It seems like it might work. Just worried about the amount of space these accounts will need.

Reply:
Updated the pr to include this. thoughts?

Reply:
Seems good to me.